src/sd.c

```diff
@@ -384,4 +393,3 @@ bool sd_format(void)
     uint8_t work[FF_MAX_SS] = {0};
     return f_mkfs("SD", &params, work, sizeof(work)) == FR_OK;
```

Comment:
What is FF_MAX_SS, and what does this param do in general?

Is this suitable for production or was it possibly only a good value for the tests/simulator? Asking b/c I am not familiar, but the sd_format function was originally added for unit tests only.

Reply:
> This set of options defines the extent of sector size used for the low level disk I/O interface, disk_read and disk_write function. Valid values are 512, 1024, 2048 and 4096. FF_MIN_SS defines minimum sector size and FF_MAX_SS defines the maximum sector size. Always set both 512 for memory card and harddisk.
 
Taken from FatFs configurations, http://elm-chan.org/fsw/ff/doc/config.html#max_ss.  Only thing that boggled me was "SD" parameter there, but I did not encounter any issue while testing out. The card resetted and continued to work without an error.